src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -81,6 +81,10 @@ public interface IAppConfig
         /// </summary>
         string Claims { get; }
 
+        /// <summary>
+        /// </summary>
+        bool IsBrokerEnabled { get; }
```

Comment:
Not sure how we want to hide this. I'm using it for unit tests, so i might have to remove those if this isn't available.

Reply:
leaving. discussed w/henry.

---
In reply to: [256601587](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r256601587) [](ancestors = 256601587)